test/functional/services/remote/webdriver.ts

```diff
@@ -100,6 +104,30 @@ async function attemptToCreateCommand(log: ToolingLog, browserType: Browsers) {
           .setFirefoxOptions(firefoxOptions)
           .setFirefoxService(new firefox.ServiceBuilder(geckoDriver.path).enableVerboseLogging())
           .build();
+
+      case 'ie':
```

Comment:
@LeeDr maybe we need to add platform check here to make sure it is run only on Windows?

```
if (process.platform !== 'win32') {
    throw new Error('Running Internet Explorer is supported only on Windows platform');
}
```

Reply:
I think this came up in another spot.  I'd like to reserve the possibility to run the tests from non-Windows OS with IE on a Windows VM.